Formula/znapzend.rb

```diff
@@ -20,6 +21,49 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    (var/"log/#{name}").mkpath
```

Comment:
We generally do not want to use `name` like this in formulae and just hard code it.

Reply:
I have no strong opinion on that.  
Just for clarification, would you say the plist should be changed in the same way?

Reply:
yes